### PR TITLE
Pull our OpenCV in install_deps

### PR DESCRIPTION
**Pull request by mhuzai**
We need to use our instrumented OpenCV instead of the vanilla one.

### CODE REVIEW

**Review by charmoniumQ — DISMISSED** [review]

**Review by apache8080 — DISMISSED** [review]

**Review by charmoniumQ — APPROVED** [review]

File: scripts/install_opencv.sh [hunk 1]
```diff
@@ -14,4 +14,9 @@ cmake \
 	-D WITH_OPENGL=ON \
 	-D OPENCV_EXTRA_MODULES_PATH="${temp_dir}/opencv_contrib/modules"
 sudo make -C "${temp_dir}/opencv/build" "-j$(nproc)" install
+
+# Load new library paths
 sudo ldconfig -v
+
+# Remove temp cmake files
+rm -f a.out cmake_hdf5_test.o
```
Comment (apache8080) [inline scripts/install_opencv.sh:22]: Why is this necessary? Don't these files only appear if cmake failed.

Comment (mhuzai) [reply scripts/install_opencv.sh:22]: These files are generated when `cmake` tests whether the compiler works or not. We didn't worry about them previously because we were invoking `cmake` from `/tmp/ILLIXR_deps` but now we invoke `cmake` from the main ILLIXR directory. `cmake` really should be cleaning up these files by itself but I couldn't find any documentation on this.

Comment (charmoniumQ) [reply scripts/install_opencv.sh:22]: Seems reasonable.

**Review by charmoniumQ — APPROVED** [review]

**Review by apache8080 — APPROVED** [review]
